src/Host/Client/Impl/rtvs/R/util.R

```diff
@@ -53,6 +53,18 @@ toJSON <- function(obj) {
   call_embedded("toJSON", obj)
 }
 
+create_blob <- function(obj) {
+    call_embedded("create_blob", obj)
+}
+
+get_blob <- function(blob.id) {
```

Comment:
We use underscore instead of dot for identifiers in R code as a matter of style (even though some existing code doesn't follow this). This is because S3 objects use dots in names to identify methods (e.g. `blob.id` would be implementation of S3 generic method `blob` for class `id`), so there's potential for confusion there.